portfolio.js

```diff
@@ -127,3 +127,45 @@ form.addEventListener('submit', (e) => {
     errorMessage.style.display = 'none';
   }
 });
+
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+      // everything except Firefox
+      e.code === 22
+      // Firefox
+      || e.code === 1014
+      // test name field too, because code might not be present
+      // everything except Firefox
+      || e.name === 'QuotaExceededError'
+      // Firefox
+      || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+      // acknowledge QuotaExceededError only if there's something already stored
+      && (storage && storage.length !== 0);
```

Comment:
- [ ] [OPTIONAL] It will be nice if you remove all these comments make your code cleaner in your repo 😉

Reply:
I totally confused on what you expect me to change. May you pleas help me

Reply:
Sure! your project is working fine, just add more commits 😉 and as optional, you can remove the comments line in your JS code " // ".

Reply:
okay thanks

Reply:
You're welcome, remainder: at least 5 commits (for this project) would be nice 💯

Reply:
Thanks a lot for the constructive review man.

Reply:
You're welcome, nice work! 💪